osi_object.proto

```diff
@@ -950,5 +954,46 @@ message MovingObject
                 BRAKE_LIGHT_STATE_STRONG = 4;
             }
         }
+
+        // Definition of vehicle roles.
+        //
+        enum Role
+        {
+            // Role of vehicle is unknown (must not be used in ground truth).
+            //
+            ROLE_UNKNOWN = 0;
+
+            // Other (unspecified but known) role of vehicle.
+            //
+            ROLE_OTHER = 1;
+
+            // The vehicle role is ambulance.
+            //
+            ROLE_AMBULANCE = 2;
+
+            // The vehicle role is civil.
```

Comment:
**Output CCB 16.02.22:**
1. Please provide better documentation for "ROLE_CIVIL"